daprdocs/content/en/concepts/dapr-services.md

```diff
@@ -0,0 +1,45 @@
+---
+type: docs
+title: "Dapr Services"
```

Comment:
This is meant to address issue #378 and so should be a "How To" document:
1. Please move this from concepts (which is a section that provides general description rather than explicit instructions)  I recommend as a new page under the reference section
2. Please rename this "Dapr Services" does not convey what this doc is about

Reply:
I think there should be a "Dapr services" page that is a directory, and then separate pages for daprd, sentry, placement, etc.

Reply:
So keep this page as "Dapr services", move it under "References", and then take all the Daprd content and put it into a new page "daprd"

Reply:
@AaronCrawfis should new page "daprd" be under concepts?

Reply:
After discussing with @AaronCrawfis: We'll be adding a concepts page that explains the various services in a future PR. For now lets keep this a daprd focused docs that is more about how to use it and when. Therefore, it should live under reference